src/shared/components/case-editor/services/wizard-page-field-to-case-field.mapper.ts

```diff
@@ -0,0 +1,84 @@
+import { WizardPageField } from '../domain';
+import { CaseField } from '../../../domain';
+import { ComplexFieldMask } from '../domain/wizard-page-field-complex-mask.model';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class WizardPageFieldToCaseFieldMapper {
+
+  mapAll(wizardPageField: WizardPageField[], caseFields: CaseField[]): CaseField[] {
+    return wizardPageField.map((wizardField: WizardPageField) => {
+      return this.map(wizardField, caseFields);
+    });
+  }
+
+  private map(wizardPageField: WizardPageField, caseFields: CaseField[]): CaseField {
```

Comment:
can we rename `wizardPageField` to `wizardField`?

Reply:
Not much value in renaming. I will keep it as it is.